docs/bundles/plain.md

```diff
@@ -29,16 +60,125 @@ manifests
+  a [BundleDeployment](https://github.com/operator-framework/rukpak#bundledeployment). Essentially, any file that would not
+  successfully `kubectl apply` will result in an error, but multi-object YAML files, or JSON files, are fine. There will
+  be validation tooling provided that can determine whether a given artifact is a valid bundle.
+
+## Building a plain bundle
+### Prerequisites
+
+In order to create a plain bundle for Rukpak, ensure your Kubernetes manifests are in a flat directory at the root of your project called `manifests/`. This allows the contents to be sourced and unpacked by the [plain provisioner](../provisioners/plain.md). It should look similar to:
+
+```bash
+$ tree manifests
+manifests
+├── namespace.yaml
+├── service_account.yaml
+├── cluster_role.yaml
+├── cluster_role_binding.yaml
+└── deployment.yaml
+```
+
+If you are using [kustomize](https://kustomize.io/) for building your manifests from a template, redirect the output into a single file under the `manifests/` directory. For example:
+```bash
+$ tree templates
+templates
+├── namespace.yaml
+├── service_account.yaml
+├── cluster_role.yaml
+├── cluster_role_binding.yaml
+├── deployment.yaml.yaml
+└── kustomization.yaml
+```
+
+```bash
+kustomize build templates > manifests/manifests.yaml
+```
+
+### Git source
+
+For using the bundle with a [git source](../sources/git.md), commit your `manifests/` directory to your git repository.
+
+### HTTP source
+
+For using the bundle with a [HTTP source](../sources/http.md), create a `.tgz` file that contains the `manifests/` directory and it's manifests. This `.tgz` file should be served when hitting the HTTP endpoint.
+
+### Image source
+
+For using the bundle with an [image source](../sources/image.md), follow the below steps:
+
+1. Create a Dockerfile at the root of the project
+```bash
+touch Dockerfile.plainbundle
+```
+
+2. Edit the Dockerfile to include the following:
+```Dockerfile
+FROM scratch
+COPY manifests /manifests
+```
+
+> Note: The Dockerfile can have any `FROM ...` directive, but it is recommended to use the `FROM scratch`
+> directive to keep the resulting image size as minimal as possible
+
+3. Build the image using a container tool like docker or podman. Use an image tag that references a repository that you have push access to. For example,
+
+```bash
+docker build -f Dockerfile.plainbundle -t quay.io/operator-framework/rukpak:example .
+```
+
+4. Push the image to the remote registry
+
+```bash
+docker push quay.io/operator-framework/rukpak:example
+```
+
+If you are looking to use a private image registry for sourcing your bundle content, see the ["Private image registries" section of the image source documentation](../sources/image.md#private-image-registries)
+
+### Upload source
+
+For using the bundle with an [upload source](../sources/upload.md), follow the below steps:
+
+1. Download the `rukpakctl` CLI tool
+```bash
+go install github.com/operator-framework/rukpak/cmd/rukpakctl
```

Comment:
```suggestion
$ go install github.com/operator-framework/rukpak/cmd/rukpakctl
```

Reply:
addressed in https://github.com/operator-framework/rukpak/pull/642/commits/486faf8915e2f3f891ee769b99651e3039d052b1